### DISCUSSION TITLE
Replacing multiple parts of text node

### DISCUSSION THREAD

**Discussion by GorroRojo** [syntax-tree/unist#109] [Q&A]
I have a text node with thevalue of "We'll be welcoming @ Pepe and @ Tata" and I'm matching something akin to `/@\S+/g` with `h('a.mention', { href: '/' + user }, '@' + user)`, findAndReplace only seems to work on the first result of each node, in this case @ Pepe, but not on the second. Is this expected behaviour? Is this what the docs mean by not doing partial matches? I looked on the web and on other questions but couldn't find the answer.

Thank you! Great package!

**Comment by wooorm** [comment]
a) you’re mentioning actual humans here
b) can you provide more code? this should work but your code is super minimal
